zuul.d/projects.yaml

```diff
@@ -6,8 +6,8 @@
     vars:
       cifmw_operator_build_golang_ct: "docker.io/library/golang:1.20"
       cifmw_operator_build_golang_alt_ct: "quay.rdoproject.org/openstack-k8s-operators/golang:1.20"
-      cifmw_tempest_tempestconf_profile:
-          overrides:
+      cifmw_test_operator_tempest_config:
```

Comment:
this doesn't seem right, you want to replace the variable meant for tempestconf, but cifmw_test_operator_tempest_config is the config for the whole operator, see it's default:
https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/test_operator/defaults/main.yml#L45

Reply:
Done, the correct var is `cifmw_tempest_tempestconf_config`.